rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -149,10 +148,14 @@ def _load_vocabulary_params(self) -> None:
             if self.OOV_words:
                 self.OOV_words = [w.lower() for w in self.OOV_words]
 
-        # Additional vocabulary size to be kept reserved
-        self.additional_vocabulary_size = self.component_config[
-            "additional_vocabulary_size"
-        ]
+        for _, value in self.component_config["additional_vocabulary_size"].items():
+            if value:
+                rasa.shared.utils.io.raise_deprecation_warning(
+                    "The parameter `additional_vocabulary_size` has been deprecated "
+                    "and you don't have to specify it anymore"
```

Comment:
Please refer to my suggestion [here](https://github.com/RasaHQ/rasa/pull/8771#discussion_r644026905) about the message that you can output.

Reply:
Also, now that you are iterating over the keys anyways you can include them in the message as well to be explicit about which attribute inside `additional_vocabulary_size` are you referring to..